docs/products/runs/private-repos.md

```diff
@@ -5,7 +5,7 @@ Grid Runs offers to solutions for using private repos:
 2. Grant Grid access to your private GitHub repos
 
 ## Grid Run localdir Option
-Grid has a good Github integration to allow running code from public or private repositories. This feature enables users to run scripts from an arbitrary local directory, even if the code is not hosted on Github. The main benefit of this feature is for users that do not need to grant Grid access to their GitHub accounts. Below is an example usage of the grid run --local dir option.
+Currently, Grid has a good Github integration to allow running code from public or private repositories. We do not currently have integration support with other code repo providers like BitBucket, Gitlab, etc. We provide the --localdir feature within Grid run to allow users to run scripts from an arbitrary local directory, regardless of where that code is hosted. The main benefit of this feature is for users that do not need to grant Grid access to their code repository accounts. Below is an example usage of the grid run --local dir option.
```

Comment:
+1 what alex said. update "good Github integration" to "native Github integration" and update "we do not currently have integration support with other" to "There is currently no support for integration with other code repository providers"

Reply:
done.